src/main/java/io/github/syst3ms/skriptparser/expressions/ExprAnswer.java

```diff
@@ -0,0 +1,60 @@
+package io.github.syst3ms.skriptparser.expressions;
+
+import io.github.syst3ms.skriptparser.Parser;
+import io.github.syst3ms.skriptparser.effects.EffAsk;
+import io.github.syst3ms.skriptparser.lang.Expression;
+import io.github.syst3ms.skriptparser.lang.TriggerContext;
+import io.github.syst3ms.skriptparser.parsing.ParseContext;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Retrieve the last input of the user in a {@link EffAsk} effect.
+ * This will essentially return the last line of the console.
+ *
+ * @name Last Answer
+ * @type EXPRESSION
+ * @pattern [the] [last] (answer|response)
+ * @since ALPHA
+ * @author ItsTheSky
+ */
+public class ExprAnswer implements Expression<String> {
+
+    static {
+        Parser.getMainRegistration().addExpression(
+                ExprAnswer.class,
+                String.class,
+                true,
+                "[the] [last] (answer|response)"
+        );
+        ANSWERS = new HashMap<>();
```

Comment:
> Methods, fields and local variables use camelCase

see [skript-parser code conventions](https://github.com/SkriptLang/skript-parser/blob/master/code-conventions.md) in root of the project